src/utils/network.ts

```diff
@@ -124,8 +124,15 @@ export const createBlockId = (): AxiosPromise =>
         })
     );
 
-export const getBlockRequest = (id: string): AxiosPromise =>
-    axios(requestOptions("GET", `${config.blockRegistry.host}/blocks/${id}`));
+export const getBlockRequest = (id: string, version?: number): AxiosPromise =>
+    axios(
+        requestOptions(
+            "GET",
+            `${config.blockRegistry.host}/blocks/${id}${
+                version ? "?version=" + version : ""
```

Comment:
Suggestion: This would read more clearly if you extracted this ternary statement to a variable.